app/assets/javascripts/tooltips.js

```diff
@@ -0,0 +1,12 @@
+jQuery(function($) {
+  var tooltipTriggers = $('.advanced-tooltip-trigger');
+  tooltipTriggers.each(function (index, el) {
+    var tooltip = $($(el).data('tooltip-target'));
+    $(el).mouseover(function () {
+      var top = $(this).offset().top - $(window).scrollTop();
+      tooltip.css({'opacity': 1, 'visibility': 'visible', 'top': top});
+    }).mouseout(function () {
```

Comment:
This is only triggered by mouse events which is fine for the setup outlined in the LSG. In the change password view however, we have an input so users might want to use TAB to switch between the form elements. Doing so will not trigger the tooltip, though.